pkgs/by-name/pr/prefect/package.nix

```diff
@@ -0,0 +1,162 @@
+{
+  lib,
+  python3,
```

Comment:
You should use python3Packages here.

Reply:
This hasn't been addressed

Reply:
My bad, let me have a look at it tonight.